.eslintignore

```diff
@@ -1 +1,3 @@
 /cli/assets
+/shell/adapter/.pnp.js
+/shell/src/D2App/*
```

Comment:
Why ignore this?

Reply:
It is ignored in: https://github.com/dhis2/app-platform/pull/219/files#diff-9421058e460d7aad16d508b9a3052acdL1

And since ESLint only looks at the `.eslintignore` in the [current working directory](https://eslint.org/docs/user-guide/command-line-interface#ignoring-files) I figured that it should really be in the top-level.

Perhaps that was mistake?

After some changes in d2-style we don't need to ignore the `.pnp.js` file either.

Reply:
I'm pretty sure this needs to be in the shell itself, since when we're running the CRA shell app that **is** the current working directory.

Reply:
(CRA uses ESLint actively, so it needs to be in the shell we copy into the application's .d2/shell directory)

Reply:
Aight, I'll move it back.

Reply:
faff521